lib/data_migrate/config.rb

```diff
@@ -22,12 +22,20 @@ def initialize
       @data_template_path = DEFAULT_DATA_TEMPLATE_PATH
       @db_configuration = nil
       @spec_name = nil
+      @test_support_enabled = false
```

Comment:
Nitpick:

I've read your PR a few times so far. Decided to do one last round from the "does it make me confused if I jump straight into the code without reading the context?". So please consider my comments as ideas / suggestions, and not something authoritative.

In this instance, I feel like the name is a bit confusing. Because it's not that we turn on _test support_, but we turn on _test generation_. So I'd prefer to have `test_generator_enabled`, `generate_tests`, `generate_specs`, `generate_test_files`, etc.

Reply:
I like this suggested change as it better represents what the configuration value achieves 💯